coa/pkg/apis/v1alpha2/providers/config/config.go

```diff
@@ -23,6 +23,6 @@ type IConfigProvider interface {
 }
 
 type IExtConfigProvider interface {
-	Get(object string, field string, overrides []string, localContext interface{}) (interface{}, error)
-	GetObject(object string, overrides []string, localContext interface{}) (map[string]interface{}, error)
+	Get(ctx context.Context, object string, field string, overrides []string, localContext interface{}) (interface{}, error)
```

Comment:
We only have configs-manager as IExtConfigProvider's implementation? right?

Reply:
yes, all three interfaces has updated context.